charts/kube-prometheus-stack/templates/crds/job.yaml

```diff
@@ -0,0 +1,73 @@
+{{- if .Values.crds.enabled }}
+{{- $appVersion := .Chart.AppVersion }}
+{{- $crds := list "alertmanagerconfigs" "alertmanagers" "podmonitors" "probes" "prometheusagents" "prometheuses" "prometheusrules" "servicemonitors" "thanosrulers" -}}
```

Comment:
Can we convert this to a config value inside the new crds key? This way users can be granular on what CRDs to manage.